tests/unittest/PinAssignment/PackagePinsLoader_test.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2022 The Foedag team
+
+GPL License
+
+Copyright (c) 2022 The Open-Source FPGA Foundation
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "PinAssignment/PackagePinsLoader.h"
+
+#include <filesystem>
+
+#include "PinAssignment/PackagePinsModel.h"
+#include "gtest/gtest.h"
+using namespace FOEDAG;
+
+TEST(PackagePinsLoader, LoadGeneral) {
+  PackagePinsModel model;
```

Comment:
We should start using setup, teardown and fixtures as applicable in this case with LoadAllPins.